### ISSUE TITLE
HDF5-1.10.1 requires munge as dependency

### ISSUE THREAD

**Issue by vanzod** [easybuilders/easybuild-easyconfigs#6655]
While building `HDF5-1.10.1-foss-2017b.eb` I stumbled into the following error:
```
/accre/arch/easybuild/software/Compiler/GCCcore/6.4.0/binutils/2.28/bin/ld.gold: error: cannot find -lmunge
```
This was due to the fact that on the specific node there was no `munge-devel` RPM installed.

We should consider adding it as an `osdependencies`.

**Comment by akesandgren** [comment]
Ahh, it's the MPI part of HDF5 that needs it due to OpenMPI with slurm.
I.e., it comes from OpenMPI/2.1.1/lib/libmpi.la

**Comment by boegel** [comment]
So, this is site-specific?

**Comment by akesandgren** [comment]
Yeah pretty much. You need munge-dev installed to build slurm and to build openmpi with slurm and so on.
